src/pages/ReportAddApproverPage.tsx

```diff
@@ -0,0 +1,205 @@
+                : approvers;
+
+        const data = sortAlphabetically(filteredApprovers, 'text', localeCompare);
+        return [
+            {
+                title: undefined,
+                data,
+                shouldShow: true,
+            },
+        ];
+    }, [employeeList, debouncedSearchTerm, countryCode, localeCompare, report.managerID, personalDetails, selectedApproverEmail, translate]);
+
+    const shouldShowListEmptyContent = !debouncedSearchTerm && !sections.at(0)?.data.length;
+
+    const addApprover = useCallback(() => {
+        const employeeAccountID = allApprovers.find((approver) => approver.login === selectedApproverEmail)?.value;
+        if (!selectedApproverEmail || !employeeAccountID) {
+            return;
+        }
+        addReportApprover(report, selectedApproverEmail, Number(employeeAccountID), currentUserDetails.accountID);
+        Navigation.goBack(ROUTES.REPORT_WITH_ID.getRoute(report.reportID));
+    }, [allApprovers, selectedApproverEmail, report, currentUserDetails.accountID]);
+
+    const button = useMemo(() => {
+        return (
+            <FormAlertWithSubmitButton
+                isDisabled={!selectedApproverEmail}
+                buttonText={translate('common.save')}
+                onSubmit={addApprover}
+                containerStyles={[styles.flexReset, styles.flexGrow0, styles.flexShrink0, styles.flexBasisAuto]}
+                enabledWhenOffline
+                shouldBlendOpacity
+            />
+        );
+    }, [addApprover, selectedApproverEmail, styles.flexBasisAuto, styles.flexGrow0, styles.flexReset, styles.flexShrink0, translate]);
+
+    const toggleApprover = useCallback(
+        (approver: SelectionListApprover) =>
+            debounce(() => {
+                if (selectedApproverEmail === approver.login) {
+                    setSelectedApproverEmail(undefined);
+                    return;
+                }
+                setSelectedApproverEmail(approver.login);
+            }, 200)(),
+        [selectedApproverEmail],
+    );
+
+    const headerMessage = useMemo(() => (searchTerm && !sections.at(0)?.data?.length ? translate('common.noResultsFound') : ''), [searchTerm, sections, translate]);
+
+    const listEmptyContent = useMemo(
+        () => (
+            <BlockingView
+                icon={Illustrations.TurtleInShell}
+                iconWidth={variables.emptyListIconWidth}
+                iconHeight={variables.emptyListIconHeight}
+                title={translate('workflowsPage.emptyContent.title')}
+                subtitle={translate('workflowsPage.emptyContent.approverSubtitle')}
+                subtitleStyle={styles.textSupporting}
+                containerStyle={styles.pb10}
+                contentFitImage="contain"
+            />
+        ),
+        [translate, styles.textSupporting, styles.pb10],
```

Comment:
Yeah like this design here is now going to be in 3 places 🤷‍♂️ 

https://github.com/Expensify/App/blob/f8cd46e879bba16e5d8b11bbb9c5b10dd89dc4b5/src/pages/workspace/workflows/approvals/WorkspaceWorkflowsApprovalsExpensesFromPage.tsx#L211C5-L225C7

https://github.com/Expensify/App/blob/f8cd46e879bba16e5d8b11bbb9c5b10dd89dc4b5/src/pages/workspace/workflows/approvals/WorkspaceWorkflowsApprovalsApproverPage.tsx#L248C1-L262C7